README.md

```diff
@@ -20,16 +20,19 @@ var after = new mapboxgl.Map({
   style: 'mapbox://styles/mapbox/dark-v9'
 });
 
-new mapboxgl.Compare(before, after, {
+// A selector or reference to HTML element
+var container = '#comparison-container';
+
+new mapboxgl.Compare(before, after, container, {
   mousemove: true, // Optional. Set to true to enable swiping during cursor movement.
-  orientation: 'vertical' // Optional. Sets the orientation of swiper to horizontal or vertical, defaults to vertical
+  orientation: 'vertical', // Optional. Sets the orientation of swiper to horizontal or vertical, defaults to vertical,
```

Comment:
This is an opinionated change I think you should remove.

Reply:
@tristen what is, the trailing comma?

Reply:
Yes.

Reply:
no problem,